internal/app/machined/pkg/runtime/v1alpha1/platform/container/container.go

```diff
@@ -17,10 +17,19 @@ import (
 	"github.com/talos-systems/talos/internal/app/machined/pkg/runtime"
 	"github.com/talos-systems/talos/internal/app/machined/pkg/runtime/v1alpha1/platform/errors"
 	"github.com/talos-systems/talos/pkg/machinery/resources/network"
+	runtimeres "github.com/talos-systems/talos/pkg/machinery/resources/runtime"
 )
 
 // Container is a platform for installing Talos via an Container image.
-type Container struct{}
+type Container struct {
+	metadata *MetadataConfig
```

Comment:
```suggestion
	*MetadataConfig
```

Reply:
I'm not sure if an embedded struct makes sense :thinking:

Reply:
I thought about it. The last opinion was - the platform can have an extra state witch we can store here (and do not mix all values together). 

I use the pointer because it is easiest way to understand metadata was initialised or not.